Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -312,7 +312,7 @@ def uninstall_script(directives, directive_name: :script, force: false, command:
       end
 
       def uninstall_pkgutil(*pkgs, command: nil, **_)
-        ohai "Uninstalling packages:"
+        ohai "Uninstalling packages (your password may be necessary):"
```

Comment:
```suggestion
        ohai "Uninstalling packages; your password may be necessary:"
```

to follow the same style as the existing message.